.gitignore

```diff
@@ -67,10 +67,11 @@ public/critical.css
 packages/volto/data
 
 # Sphinx and MyST
-docs/_build/
+/_build
+/docs/_build
```

Comment:
Does this directory exist after `make clean docs-html`? If not, then let's remove it from here.

Reply:
It does not, removing...